### PR TITLE
Login

### PR DESCRIPTION
**Pull request by mihaiberq**
- [x] Implement Redux for token management
- [x] Redirect user after login
- [ ] Prompt user to login after signup

### CODE REVIEW

File: src/components/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export { default as ButtonWrapper } from './ButtonWrapper';
+export { default as LoginForm } from './LoginForm';
+export { default as LoginWrapper } from './LoginWrapper';
+export { default as SignupWrapper } from './SignupWrapper';
```
Comment (vladstoick) [inline src/components/Login/index.js]: dc faceti asa?

Comment (vladstoick) [inline src/components/Login/index.js]: `export SignupWrapper from './SignupWrapper'`

Comment (vladstoick) [inline src/components/Login/index.js]: si pe urma sa fie folosit `import { SignupWrapper } from './Login'`

Comment (mihaiberq) [inline src/components/Login/index.js]: imi apare si la importuri absolute - ma uit pe un issue si cica e de la webpack

Comment (vladstoick) [inline src/components/Login/index.js]: fa cum ti-am zis

Comment (mihaiberq) [inline src/components/Login/index.js]: image

**Pull request merged**